### PR TITLE
[11.0][MIG] website_crm_quick_answer

### PR DESCRIPTION
**Pull request by wpichler**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

File: website_crm_quick_answer/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    'version': '11.0.1.0.0',
+    'depends': [
+        'base_automation', 'website_crm',
+    ],
+    'data': [
+        "data/automation.xml",
+    ],
+    'author': 'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://www.tecnativa.com',
```
Comment (MeetKD) [inline website_crm_quick_answer/__manifest__.py]: website should be `https://github.com/OCA/website` and you can also remove th4e encoding from py files.

**Review by MeetKD — CHANGES_REQUESTED** [review]

**Comment by rafaelbn** [comment]
Hello @wpichler !

Are you going to continue with this PR? If not please close this PR.

We need to migrate this module to v11 so if no answer we will do a new PR

You haven't answered to last comments

Thanks a lot!

File: website_crm_quick_answer/README.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+=====
+
+To use this module, you need to:
+
+* Have an email address in your main company. It will be used as sender
+  address.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/186/10.0
```
Comment (nikul-serpentcs) [inline website_crm_quick_answer/README.rst]: @wpichler IMO Change target `11.0`

File: website_crm_quick_answer/data/automation.xml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (nikul-serpentcs) [inline website_crm_quick_answer/data/automation.xml:2]: IMO `<odoo noupdate="1">` and remove [this](https://github.com/OCA/website/pull/473/files#diff-f8cf199c897affda1478dd1864273340R3)

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

File: website_crm_quick_answer/README.rst (hunk 2 above)
Comment (wpichler) [reply website_crm_quick_answer/README.rst]: how to create a new runbot target ?

File: website_crm_quick_answer/data/automation.xml (hunk 3 above)
Comment (wpichler) [reply website_crm_quick_answer/data/automation.xml:2]: done

File: website_crm_quick_answer/README.rst (hunk 2 above)
Comment (nikul-serpentcs) [reply website_crm_quick_answer/README.rst]: only change `:target: https://runbot.odoo-community.org/runbot/186/11.0`

File: website_crm_quick_answer/__manifest__.py (hunk 1 above)
Comment (wpichler) [reply website_crm_quick_answer/__manifest__.py]: changed website now

File: website_crm_quick_answer/README.rst [hunk 4]
```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (simahawk) [inline website_crm_quick_answer/README.rst]: png

**Review by simahawk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**